back-end/api/controllers/star_controller.js

```diff
@@ -0,0 +1,34 @@
+export default class StarController {
+    static #inputBoundary;
+    static #outputBoundary;
+
+    static setTranscriptInteractor(interactor) {
+        if(interactor.isInputBoundaryInterface){
+            this.#inputBoundary = interactor;
+        } else {
+            throw new Error("not an InputBoundary");
+        }
+    }
+
+    static setOutputBoundary(outputBoundary) {
+        if(outputBoundary.isOutputBoundaryInterface){
+            this.#outputBoundary = outputBoundary;
+        } else {
+            throw new Error("not an OutputBoundary");
+        }
+    }
+
+    static async putStar(req, res, next) {
+        try {
+            const question = req.body.question;
+            const projectId = req.body.projectId;
+            const starStatus = req.body.status;
+            await this.#inputBoundary.setStarStatus({question: question, project_id: projectId},{star: starStatus})
```

Comment:
Shouldn't we set the star status to !req.body.status?

Reply:
Yeah this will change once I'm sure what is being sent back. It might be the value we want star to be or current value.

Reply:
Sounds good :+1:.